dimod/__init__.py

```diff
@@ -62,3 +62,6 @@
 import dimod.utilities
 
 from dimod.vartypes import *
+
+# flags for some global features
+REAL_INTERACTIONS = False
```

Comment:
How about you make this a global default for QM's class variable that can be overridden in an instance?

Reply:
no class variables on extension classes unfortunately, cython limitation.